documentation/commands/git-town-prepend.md

```diff
@@ -0,0 +1,42 @@
+#### NAME
+
+git-town-prepend - create a new feature branch between the current branch and its parent
+
+
+#### SYNOPSIS
+
+```
+git town-prepend <branch_name>
+git town-prepend (--abort | --continue)
+```
+
+
+#### DESCRIPTION
+
+Syncs the given parent branch (default: main branch),
```

Comment:
There is no given parent branch, the parent from the current branch is used (and would be prompted for it unknown)